lib/jekyll-github-metadata/value.rb

```diff
@@ -3,6 +3,9 @@
 module Jekyll
   module GitHubMetadata
     class Value
+      extend Forwardable
+      def_delegators :render, :+, :to_s, :to_json, :eql?
```

Comment:
Clever. Can you memoize `render` so it doesn't render on each call? Or use `:value` instead of `:render`.

I'd also like to have a test for each of these – it can be really basic, but I just want to ensure they don't break.

Why `#+`?

Per [`Object#hash`](https://ruby-doc.org/core-2.4.1/Object.html#method-i-hash), if you specify `#eql?` you must *also* specify `#hash`:

> Generates an Integer hash value for this object. This function must have the property that `a.eql?(b)` implies `a.hash == b.hash`.